src/wallet/test/fuzz/coinselection.cpp

```diff
@@ -144,6 +155,21 @@ FUZZ_TARGET(coinselection)
             assert(result->GetWeight() > weight);
         }
     }
+
+    std::vector<COutput> manual_utxos;
+    CAmount manual_balance{CreateCoins(fuzzed_data_provider, manual_utxos, coin_params)};
+    if (manual_balance <= 0) return;
```

Comment:
can't be negative. It's a ranged amount.

Reply:
Done